recipe/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
```

Comment:
I don't know if they have any special options, but I try to make an effort of skimming `CMakeLists.txt` to see some of the options available and what is pertinent.